skimage/morphology/isotropic.py

```diff
@@ -57,7 +57,7 @@ def isotropic_erosion(image, radius, out=None, spacing=None):
 def isotropic_dilation(image, radius, out=None, spacing=None):
     """Return binary morphological dilation of an image.
 
-    This function returns the same result as :func:`skimage.morphology.binary_dilation`
+    This function returns the same result as :func:`skimage.morphology.dilation`
```

Comment:
If it actually returns the same result as `binary_dilation`, when changing to `dilation`, I guess we need to include the same warning that differentiates `dilation` from `binary_dilation`, i.e., "Note the lack of mirroring for non-symmetric footprints (see docstring notes)."

Reply:
Maybe, after the 'Returns' section:
```py
    Notes                                                                      
    -----
    For non-symmetric footprints, :func:`skimage.morphology.dilation`
    and this function produce an output that differs:
    This function mirrors the footprint, whereas `dilation` does not.
    :func:`skimage.morphology.mirror_footprint` is available to correct for this.
```

Reply:
See general discussion in https://github.com/scikit-image/scikit-image/pull/7665#discussion_r1929768123.